frontend/src/components/ProjectCard.vue

```diff
@@ -0,0 +1,59 @@
+<template lang="pug">
+  v-card.pa-3(:min-height="getProjectHeight" :width="1.1*getProjectHeight"  border hover color="#272727")
+    div(align="center")
+      h2(align-text="center" style="color:#ffffff" underline)
+        span CDC Portal (3

```

Comment:
Keep this data in an array like for Profile. Do it for the rest

Reply:
This will be done with completion of Profile card(upcoming PR).

Reply:
This will be done with completion of Project card(upcoming PR).